README.md

```diff
@@ -98,6 +96,9 @@ Set to `true` to run parameter validations without actually starting a run in Ra
 #### Default behavior
 If no `dry_run` parameter is passed in, the run will be started in Rainforest.
 
+## Rerunning failed tests
+The Rainforest QA GitHub Action uses GitHub [caching](https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows) to know when a workflow is [being rerun](https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs). It will automatically rerun only the tests which failed in the previous run.
```

Comment:
I think this could do with a bit more context about why to do this and how we expect users to use it - we should give an opinion about how they use the action in their workflows. Not sure if this context belongs in this PR or in this part of the README.

Reply:
Good point 👍🏼 For the orb, that context was hidden away in the [blog post](https://rainforest.engineering/2021-01-20-shipping-faster-orb/) detailing how to use the feature. I'll add a little bit of copy here to make it more obvious you shouldn't rerun failed workflows without checking results/fixing tests first.